### ISSUE TITLE
Need To Know Device Ping For Expired Devices

### ISSUE THREAD

**Issue by SmartGpsBD** [traccar/traccar#5238]
Dear Anton, Many many thanks for your great project.
when a device was expired we don't know the device is in working condition or not.
An Example: A device was expired 2 months ago and in this time the device power connection was broken.
Now an user pay 1 month subscription fee and device is now active in server side but in device side device power connection was broken.
Now this user one month subscription will be wasted.
So if our server show a ping icon in every device then customer know this device has a problem and they not make a payment for this type of device.
![pic1](https://github.com/traccar/traccar/assets/73643926/437b9c04-8022-48fc-ac91-74b03101315f)
In above picture there are two devices in expired condition. Now as a customer we don't know this devices are in active condition and make a payment then device will be in Online condition.
![pic2](https://github.com/traccar/traccar/assets/73643926/3da98ec1-921c-4974-97e1-d21e6af7920e)
In above picture we make payment for both device and "Car-01" is now online but "Car-02" is offline because of the "Car-02" physical wiring connection was broken.
So As a customer we want to know about device condition before make a payment.
Thanks a lot for your great work.

**Comment by YeiserCode** [comment]
Hi, I would like to be able to collaborate with this feature since I have already done it a couple of times for custom servers

**Comment by SmartGpsBD** [comment]
How can we help you? If you need more information please mention it here.
We will try to share more information for this tropic.
Thanks

**Comment by YeiserCode** [comment]
Hello, I've been a bit busy in other projects but if I can contribute this I'll share with you an images of how I have it implemented in other Traccar interfaces.

Use Translator

![Captura de pantalla 2024-02-21 152640](https://github.com/traccar/traccar/assets/131619978/9c122122-4325-4922-b0cd-af30d596deae)
![Captura de pantalla 2024-02-21 152723](https://github.com/traccar/traccar/assets/131619978/44cc9948-8ed9-4d69-9784-ff3a4486f7fe)
![Captura de pantalla 2024-02-21 152731](https://github.com/traccar/traccar/assets/131619978/291fd17f-4878-4a3d-82a9-cb2fc4caa8cb)

**Comment by YeiserCode** [comment]
That's right, friend Antom, that's why I took it as an example since it's a totally different website from Modern, I just want to take it as a reference if that's what you're requesting
